builders/integrations/oracles/supra.md

```diff
@@ -0,0 +1,237 @@
+---
+title: Supra Oracles
+description: Supra's Pull Oracle provides low-latency, on-demand price feed updates for a variety of use cases. Learn how to integrate Supra's oracle on Moonbeam.
+---
+
+# Supra Oracles
+
+## Introduction {: #introduction }
+
+[Supra](https://supraoracles.com){target=\_blank} is a novel, high-throughput oracle and intralayer: a vertically integrated toolkit of cross-chain solutions (data oracles, asset bridges, automation network, and more) that interlink all blockchains, public (L1s and L2s) or private (enterprises), including Moonbeam.
+
+Supra provides decentralized oracle price feeds that can be used for on-chain and off-chain use cases such as spot and perpetual DEXes, lending protocols, and payment protocols.
+
+In this guide, you'll learn how to use Supra Oracles' price feeds to fetch price data in smart contracts on Moonbeam.
+
+--8<-- 'text/_disclaimers/third-party-content-intro.md'
```

Comment:
IMHO, Is there a way to make the disclaimer smaller, like the bottom of the page? 

It doesn't have to be tiny but maybe a few sizes smaller ? 

<img width="894" alt="Screenshot 2024-02-22 at 4 28 40 PM" src="https://github.com/moonbeam-foundation/moonbeam-docs/assets/5834262/a9771e23-4977-4a17-aee3-44f9fc694d7c">